buildlib/az-distro-release.yml

```diff
@@ -62,20 +59,31 @@ jobs:
 
       - bash: |
           set -eEx
+
+          # Build
           ./autogen.sh
           ./contrib/configure-release --with-cuda --with-java=no
           make dist
           tarball=$(echo ucx*.tar.gz)
           tar -xzvf ${tarball}              # extract the sources in a subdirectory
           cd $(tar tf ${tarball} | head -1) # go to extracted tarball directory
           echo 10 > debian/compat   # https://www.debian.org/doc/manuals/maint-guide/dother.en.htmdpl#compat
-          dpkg-buildpackage -us -uc -Pcuda
+          dpkg-buildpackage -d -us -uc -Pcuda
           cd ..                             # Move back to the working directory
-          find . -name '*.deb'
-          VER="${POSTFIX#ucx-}"   # Remove 'ucx' prefix from the POSTFIX string
-          # Rename DEB files
+
+          # Rename DEB files          
+          VER="${POSTFIX#ucx-}"     # Remove 'ucx-' prefix from the POSTFIX string
           find . -name "ucx*.deb" -exec bash -c 'mv "$1" "${1%%_*}-'"${VER}"'.deb"' _ {} \;
+          find . -name '*.deb'      # Show new names
+
+          # Remove superfluous dependency
```

Comment:
Please add TODO to rework this W/A later to something more robust

Reply:
Absolutely. This is a temporary workaround.
We are planning to make our own base image that closely resembles the default client-side installation of CUDA.

Reply:
Added to the backlog.